projects/slap_manipulation/src/slap_manipulation/agents/language_ovmm_agent.py

```diff
@@ -0,0 +1,233 @@
+import logging
+from enum import Enum
+from glob import glob
+from typing import Any, Dict, List, Tuple
+
+import pandas as pd
+
+from home_robot.agent.ovmm_agent.pick_and_place_agent import (
+    PickAndPlaceAgent,
+    SimpleTaskState,
+)
+from home_robot.core.interfaces import Action, DiscreteNavigationAction, Observations
+
+
+class GeneralTaskState(Enum):
```

Comment:
do you want to move this to home_robot.core.interfaces or something? feel like it's pretty general